test/minting/AccountUtils.js

```diff
@@ -0,0 +1,185 @@
+// set to true to enable verbose logging in the tests
+const debugLogging = false;
+const assertDiff = require("assert-diff");
+assertDiff.options.strict = true;
+
+const Q = require("q");
+const clone = require("clone");
+const util = require("util");
+
+// named list of all accounts
+const Accounts = {
```

Comment:
(note: variations of this list of ganache accounts is defined in a few places in our test suite; I do plan to centralize this list in a follow-up, but would like to get this merged first since it blocks other work, and this PR is large enough without that additional test restructuring.)